lib/screens/play/play_page.dart

```diff
@@ -66,11 +66,16 @@ class PlayPage extends StatelessWidget {
                           padding: EdgeInsets.fromLTRB(10, 10, 10, 0),
                           child: Center(child: DeckStatePanel()),
                         ),
-                        SizedBox(height: 10),
+                        SizedBox(height: 5),
                         Expanded(child: CardDisplay(state.currentCard)),
                         Container(
-                          height: (MediaQuery.of(context).size.height) * 0.25,
-                          padding: EdgeInsets.fromLTRB(0, 10, 0, 10),
+                          height: (MediaQuery.of(context).size.height) * 0.05,
+                          padding: EdgeInsets.fromLTRB(0, 25, 0, 0),
+                          child: Center(child: Text('Resource Count: ' + state.playerHand.cards.length.toString())),
```

Comment:
I think we could use Dart string interpolation here to simplify a bit
```suggestion
                          child: Center(child: Text("Resource Count: ${state.playerHand.cards.length}")),
```

Reply:
Cool, I originally tried just adding the number hoping for implicit conversion (I use Scala at work) - this is nicer, similar to Scala's interpolation too. I've pushed the change.

I borrowed this originally from play_page_test.dart, where we could potentially also switch to string interpolation:
```
final deckFinder = find.widgetWithText(CardPile, 'Deck: ' + state.deck.length.toString());
final discardFinder = find.widgetWithText(CardPile, 'Discard: ' + state.discard.length.toString());
```